capellambse/extensions/reqif/elements.py

```diff
@@ -298,10 +298,17 @@ def __repr__(self) -> str:
 
     def _short_repr_(self) -> str:
         mytype = type(self).__name__
+        if mytype.startswith("Enum"):
+            value = ", ".join(self.values.by_long_name)
+        elif mytype.startswith("Date"):
+            value = helpers.strf_datevalue(self.value)
```

Comment:
IMO we shouldn't alter the value in any way, which this helper does (it converts it to UTC). We should just use `value.isoformat()` directly.